test/parallel/test-zlib-from-concatenated-gzip.js

```diff
@@ -38,3 +41,26 @@ fs.createReadStream(pmmFileGz)
     assert.deepStrictEqual(Buffer.concat(pmmResultBuffers), pmmExpected,
       'result should match original random garbage');
   }));
+
+// test that the next gzip member can wrap around the input buffer boundary
+[0, 1, 2, 3, 4, defEncoded.length].forEach((offset) => {
```

Comment:
@kthelgason I *think* testing for `0` should take care of the case where data members fall exactly on input boundaries:

The first `unzip.write()` call writes `abcEncoded` + 0 bytes from `defEncoded`, and the second `unzip.end()` call writes all of `defEncoded`. Or did you have something else in mind?